stable/pipelines/values.yaml

```diff
@@ -193,6 +193,10 @@ pipelines:
     enabled: false
   ## Logging settings
   logging:
+    ## livelog settings
+    view:
+      ## if enabled, livelogs will be enabled
+      enabled: false
```

Comment:
is this disabled by default?

Reply:
@shahiinn can you check it with @chukka and @JfrogPrasanna please?

Reply:
@rahulsadanandan default false is fine for 7.16

Reply:
It is set false now.